src/v2/api/responses/StudentResponse.ts

```diff
@@ -21,6 +22,39 @@ export class BaseStudentResponse {
     lastName: string;
 }
 
+export class SimpleStudentResponse extends BaseStudentResponse {
+  @ApiProperty({
+    description: 'Student\'s id',
+  })
+    id: string;
+
+  @ApiProperty({
+    description: 'Student\'s state',
+    enum: State,
+  })
+    state: State;
+
+  @ApiProperty({
+    description: 'Student\'s group role',
+    enum: GroupRoles,
+  })
+    role: string;
```

Comment:
In swagger it shows like number 
![image](https://github.com/fictadvisor/fictadvisor-api/assets/116638814/0fe09acf-7037-4b33-8737-17adce7cbbf3)